.storybook/preview-head.html

```diff
@@ -1,3 +1,114 @@
 <script>
   window.global = window;
-</script>
\ No newline at end of file
+</script>
+
+<style>
+/* Import fonts from Spruce so they don't need to be uploaded separately. */
+
+  /* 
+   * Euclid
+  /*
+
+  /* Semibold */
+  @font-face {
+    font-family: "Euclid Circular A";
+    font-weight: 700;
+    font-style: normal;
+    src: url("https://spruce.mongodb.com/static/fonts/EuclidCircularA/EuclidCircularA-Semibold-WebXL.woff2")
+        format("woff2"),
+      url("https://spruce.mongodb.com/static/fonts/EuclidCircularA/EuclidCircularA-Semibold-WebXL.woff")
```

Comment:
CORS doesn't allow this 
<img width="1195" alt="image" src="https://github.com/evergreen-ci/spruce/assets/4605522/1a8ea0d7-fec4-4da6-9c25-421af26dbae9">